lib/dart/pubspec.yaml

```diff
@@ -24,20 +24,15 @@ homepage: http://thrift.apache.org
 documentation: http://thrift.apache.org
 
 environment:
-  sdk: ">=1.13.0 <2.0.0"
+  sdk: '>=1.24.3<3.0.0'
```

Comment:
For consistency, should there be a space between the two version constraints? (3<3 becomes 3 <3)

Reply:
They are both valid, so I'm not super concerned but if you'd like I can add a space.

Reply:
It doesn't matter to me.